### PR TITLE
Refactor/create new meetings samples

### PR DESCRIPTION
**Pull request by InteractiveTimmy**
# Pull Request

## Description

The scope of this pull request is to implement the HTML frame needed to further migrate the functionality of the original meetings plugin samples to the new meetings samples. Note that all content within the `<section>` elements can be re-written based on the needs of each section, and the content of these areas are implemented to help provide a base shape for developers to implement against.

The sample can be viewed by running `npm run samples:serve`, then navigating to `https://localhost:8000/browser-meetings/`.

Fixes # [SPARK-142730](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-142730)

## Type of Change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: packages/node_modules/samples/browser-meetings/index.html [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+        <fieldset>
+          <legend>Current Joined Meeting</legend>
+          <input id="meetings-leave" title="leave" type="submit" value="Leave" />
+          <label>[[CURRENT MEETING IDENTIFIER]]</label>
+        </fieldset>
+      </form>
+    </section>
+
+    <section>
+      <h2>Meetings Devices</h2>
```
Comment (taymoork2) [inline packages/node_modules/samples/browser-meetings/index.html:103]: same as above comment

```suggestion
      <h2>Meeting Devices</h2>
      <blockquote>[[MEETING Devices DESCRIPTION]]</blockquote>
```

File: packages/node_modules/samples/browser-meetings/index.html [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+          <legend>Registration</legend>
+          <input id="registration-register" title="register" type="submit" value="Register" />
+          <input id="registration-unregister" title="unregister" type="submit" value="Unregister" />
+          <label id="registration-status">[[REGISTRATION STATUS]]</label>
+        </fieldset>
+      </form>
+    </section>
+
+    <section>
+      <h2>Meetings Management</h2>
```
Comment (taymoork2) [inline packages/node_modules/samples/browser-meetings/index.html:59]: I think we should describe each flow and indirectly explain why the flow is ordered this way

I still get confused by half the stuff within the kitchensink, what's it function and why it's placed there

```suggestion
      <h2>Meetings Management</h2>
      <blockquote>[[MEETING MANAGEMENT DESCRIPTION]]</blockquote>
```

File: packages/node_modules/samples/browser-meetings/index.html [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+      <form id="meetings-create">
+        <fieldset>
+          <legend>Create a Meeting</legend>
+          <input id="create-meeting-destination" name="createMeetingDestination" placeholder="Destination" value="" type="text" />
+          <input id="create-meeting-action" title="create" type="submit" value="Create" />
```
Comment (taymoork2) [inline packages/node_modules/samples/browser-meetings/index.html:64]: We should probably add back (some of) the examples
And isn't submit supposed to be a button?

File: packages/node_modules/samples/browser-meetings/index.html (hunk 2 above)
Comment (InteractiveTimmy) [reply packages/node_modules/samples/browser-meetings/index.html:59]: You know, I actually had these implemented and then saw that they were consuming too much space. I'll drop these back in.

File: packages/node_modules/samples/browser-meetings/index.html [hunk 4]
```diff
@@ -0,0 +1,367 @@
...
+          </div>
+          <br />
+          <input id="participants-update" title="updateParticipants" type="submit" value="Update Participants" />
+        </fieldset>
+      </form>
+    </section>
+    <section>
+      <h2>Meeting Streams</h2>
+      <form id="streams">
+        <div class="flex">
```
[the remark this answers is not in the document]
Comment (InteractiveTimmy) [reply packages/node_modules/samples/browser-meetings/index.html:343]: This is just a flex with wrap, I can separate it into two rows, but this means they will not be adaptive.

File: packages/node_modules/samples/browser-meetings/index.html (hunk 3 above)
Comment (InteractiveTimmy) [reply packages/node_modules/samples/browser-meetings/index.html:64]: `input="submit"` and `<button />` are very similar, and can typically be used interchangeably. I'll switch to button though, for more clarity.

File: packages/node_modules/samples/browser-meetings/index.html [hunk 5]
```diff
@@ -0,0 +1,367 @@
...
+          <input id="prm-id" name="PmrId" placeholder="PMR ID" value="" type="text" />
+          <input id="prm-pin" name="prmPin" placeholder="PMR PIN" value="" type="text" />
+          <input id="pmr-claim" title="claim" type="submit" value="Claim" />
+          <p id="pmr-details">[[PERSONAL MEETING ROOM DETAILS]]</p>
+        </fieldset>
+      </form>
+    </section>
+
+    <section>
+      <h2>Meeting Controls</h2>
```
[the remark this answers is not in the document]
Comment (InteractiveTimmy) [reply packages/node_modules/samples/browser-meetings/index.html:179]: Screen Shot 2020-05-13 at 11 42 59 AM
Screen Shot 2020-05-13 at 11 42 50 AM

File: packages/node_modules/samples/browser-meetings/index.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (InteractiveTimmy) [reply packages/node_modules/samples/browser-meetings/index.html:343]: Screen Shot 2020-05-13 at 11 42 59 AM
Screen Shot 2020-05-13 at 11 42 50 AM

**Comment by InteractiveTimmy** [comment]
> Overall I think it's really good. I suggest a small description of each section explaining the context and maybe a few examples.

I think we should definitely add examples for each section, but it should be scoped to a different JIRA. I'll be sure to create a follow up to this reorg to add examples to each section. A description should be implemented at this time though.

**Pull request merged**